test_framework.js

```diff
@@ -0,0 +1,16 @@
+const assertEquals = (expected, actual) => expected === actual;
```

Comment:
Sufficient tests were written for the purpose of the challenge. However, inefficient to create a new test framework, that does not provide a wide variety of tests, each time.
- Could make use of pre-existing test frameworks, such as Jasmine.